backend_new/README.md

```diff
@@ -131,6 +149,14 @@ Running the following will run all unit tests:
 $ yarn test
 ```
 
+
+## Testing with code coverage
+We have set up both code coverage and code coverage benchmarks. These benchmarks must be met for your PR to pass CI checks. Test coverage is calculated against both the integration and unit test runs. You can run test coverage with the followig:
```

Comment:
What are the code coverage benchmarks?

Reply:
As it stands right now:
```
coverageThreshold: {
    global: {
      branches: 75,
      functions: 90,
      lines: 90,
    },
  },
 ```
 
 are our thresholds. Those can be found in backend_new/test
/jest-with-coverage.config.js

Reply:
Sounds good! I'd love to touch base w the team about this to make sure we're all on the same page since it'd be a new flow for us, maybe at the workshop Thursday?